lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -1,7 +1,7 @@
 <% if (is_enrolled) { %>
     <div class="enrollment-info"><%- gettext('enrolled') %></div>
     <% if (is_enrollment_open || is_course_ended) { %>
-        <a href="<%- course_url %>" class="btn-neutral btn view-course-link">
+        <a href="<%- course_url %>" class="btn view-course-link">
```

Comment:
@clrux - @rlucioni and I were confused by the changes in look-and-feel of the various button classes. Why does the neutral button show brand blue on hover, but the basic button shows a washed out blue? It would seem to me that the basic button should show brand blue on hover, and that the neutral button should maybe show an accent grey.

@chris-mike do you have a UX preference for the buttons? To my eye, the default button type no longer looks as good as the others:

http://ux.edx.org/components/buttons/